public/app/plugins/datasource/prometheus/query_hints.ts

```diff
@@ -127,3 +127,28 @@ export function getQueryHints(query: string, series?: any[], datasource?: Promet
 
   return hints;
 }
+
+export function getInitHints(datasource?: PrometheusDatasource): QueryHint[] {
```

Comment:
As we use it in single place, from data source so it is always present the optionality of datasource argument looks bit off.

Reply:
Yeah, good point. I made this argument non-optional.